File: snippets/python/remove_a_tag.py
import hashlib
import mailchimp_marketing as MailchimpMarketing
from mailchimp_marketing.api_client import ApiClientError

EMAIL = "urist.mcvankab@example.com"
LIST_ID = "YOUR_LIST_ID"
SUBSCRIBER_HASH = hashlib.md5(EMAIL.encode('utf-8')).hexdigest()

client = MailchimpMarketing.Client()
mailchimp.set_config({
  "api_key": "YOUR_API_KEY",
  "server": "YOUR_SERVER_PREFIX"
})
try:
    response = client.lists.update_list_member_tags(LIST_ID, SUBSCRIBER_HASH, body={
        "tags": [{
            "name": "Influencer",
            "status": "inactive"
        }]
    })
    print("client.ping.get() response: {}".format(response))
except ApiClientError as error:
    print("An exception occurred: {}".format(error.text))

File: snippets/node/remove_a_tag.js
const listId = "YOUR_LIST_ID";
const email = "urist.mcvankab@example.com";
const subscriberHash = md5(email.toLowerCase());

async function run() {
  const response = await mailchimp.lists.updateListMemberTags(
    listId,
    subscriberHash,
    {
      body: {
        tags: [
          {
            name: "Influencer",
            status: "inactive",
          },
        ],
      },
    }
  );

  console.log(
    `The return type for this endpoint is null, so this should be true: ${
      response === null
    }`
  );
}

run();